src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,73 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
 public class WorkWithFile {
+    private static final String OPERATION_SUPPLY = "supply";
+    private static final String OPERATION_BUY = "buy";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName));
+                BufferedWriter writer = new BufferedWriter(new FileWriter(toFileName))) {
+
+            Map<String, Integer> supplyData = new LinkedHashMap<>();
+            Map<String, Integer> buyData = new LinkedHashMap<>();
+
+            String line;
+            while ((line = reader.readLine()) != null) {
+                String[] data = line.split(",");
+                String operationType = data[0];
```

Comment:
Avoid magic numbers: Indexes should always have constants

Reply:
Hi! Instead of indexes 0 and 1  were created constants:OPERATION_TYPE_INDEX = 0; AMOUNT_INDEX = 1; to avoid magic numbers. Thank you!